packages/@coorpacademy-progression-engine/src/reducers/variables.js

```diff
@@ -7,7 +7,7 @@ const variables = (config: Config) => (_state: State, action: Action): State =>
   switch (action.type) {
     case 'answer':
     case 'move': {
-      const {instructions} = action.payload;
+      const {instructions = null} = action.payload;
```

Comment:
don't understrand the aim of it ?

Reply:
you're right, no need to set this default behaviour. I had some issues typing these instructions, but somehow it work without this by now

Reply:
✅